docs/v7-migration.md

```diff
@@ -143,50 +140,234 @@ If you are converting from an older typed client to a new one you will need to m
 - The app client no longer creates or deploys contracts, you need to use the factory for that, which will in turn give you a typed client instance
 - Method calls are no longer directly on the typed client, instead they are nested via `appClient.send.` and `appClient.createTransaction.`
 - Method calls take a single params object (rather than (args, params)) and the args are nested in an `args` property within that object
-- The `compile` method returns `{compiledApprovalProgram, compiledClearStateProgram, approvalProgramCompilationResult?, clearStateProgramCompilationResult?}` rather than `{approvalCompiled, clearCompiled}`
 - `extraPages` is no longer nested within a `schema` property, instead it's directly on the method call params as `extraProgramPages`
 - `client.compose()` is now `client.newGroup()`
 - `client.compose()....execute()` is now `client.compose()....send()`
 
-### Step 5 - Rename AlgokitComposer to TransactionComposer
+## >=6.1.0 Migration Guide
+
+Assuming you have actioned the deprecation notices as part of moving to the `>=6.1.0` version and have started using the early version of the `AlgorandClient`, then you need to be aware of some breaking changes that we have made to accommodate the feature set of v7. Any migration information related to the stateless function based interface is available in the [<6.1.0 Migration Guide](#610-migration-guide).
```

Comment:
This first part of the sentence is a bit confusing

Reply:
Yeah I don't think it's needed to be honest, will refine.